### ISSUE TITLE
Tap to click on trackpad calls hold action rather than tap action

### ISSUE THREAD

**Issue by michaelblight** [Clooos/Bubble-Card#860]
For any feature request you can open a new discussion here:  
https://github.com/Clooos/Bubble-Card/discussions/categories/feature-requests

For any question you can open a new discussion here:  
https://github.com/Clooos/Bubble-Card/discussions/categories/q-a

**Describe the bug**  
A minor bug - on a laptop (or tablet with keyboard) you can enable tap to click for the trackpad. If you do this and tap, rather than calling the tap action, it calls the hold action. If you click down on the trackpad, it calls tap action as expected. The behaviour is the same on Chrome and Firefox (didn't test other browsers). I also only tested on sub-buttons.

**To Reproduce**  
1. Create a sub-button with both tap and hold actions (but different to each other).
2. Enable "Tap with a single finger to single-click"
3. Tap the sub-button and see the hold action is called rather than the tap action.
4. Fully click down with the trackpad on the sub-button and the tap action is called.

**Expected behavior**  
A tap on the trackpad should call the tap action, not the hold action.

**Screenshots**  

**YAML**  
If applicable, add any relevant YAML code.

```yaml
Your code here
```

**Informations (please complete the following information):**  
 - OS: [e.g. iOS] Windows or iOS
 - Browser/App: [e.g. chrome, safari] Chrome or Firefox
 - Bubble Card version: [e.g. 2.0.4] v2.2.4
 - Home Assistant version: [e.g. 2024.6.4] 2024.9.1

**Additional context**  

Thank you! 🍻

**Comment by Clooos** [comment]
Hi! I've just released a new beta that might fix your issue, give it a try and tell me if it's fixed 🤞

https://github.com/Clooos/Bubble-Card/releases/tag/v2.3.0-beta.3

**Comment by michaelblight** [comment]
Yes it does seem to be fixed - thanks. That version also changes the appearance of sub-buttons that are lights to indicate brightness - will you be including a changelog to list the changes? I recently changed all my dashboards to use bubble card, so am keen to keep on eye on enhancements. Great card btw!

**Comment by Clooos** [comment]
Glad that it's fixed! And what do you mean? I add a changelog for each release.

**Comment by michaelblight** [comment]
It's probably that I don't know where to look then - I assumed there'd be a CHANGELOG.md file in the root.

**Comment by Clooos** [comment]
No, I just add the changelog in the release, like here:
https://github.com/Clooos/Bubble-Card/releases/tag/v2.3.0-beta.3

**Comment by michaelblight** [comment]
Ah, many thanks. Github used to have "Releases" in the tabs at the top, but they moved it a few years ago and so I forget about that possibility.
